frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts

```diff
@@ -136,6 +142,8 @@ export class CombinedDisk {
     return null;
   };
 
+  getPVCName = () => this.pvcName;
```

Comment:
we could move this line to the implementation here
```suggestion
  getPVCName = () => this.persistentVolumeClaimWrapper.getName()|| disk.getPVCNameBySource();
```

Reply:
or just to implement the switches according to the `VolumeType` instead of `getPVCNameBySource()`